### PR TITLE
style: open `sharkdp/numbat` in new tab

### LINKED ISSUE #302: GitHub corner should open this repository in a new tab

### PR DESCRIPTION
**Pull request by hamirmahal**
fixes #302.

### CODE REVIEW

**Comment by hamirmahal** [comment]
# Without this change

[Screencast from 01-29-2024 12:33:08 AM.webm](https://github.com/sharkdp/numbat/assets/43425812/09703cda-6f69-4d18-a770-334f71fb199b)

**Comment by hamirmahal** [comment]
# With this change

[Screencast from 01-29-2024 12:34:07 AM.webm](https://github.com/sharkdp/numbat/assets/43425812/e648fde4-358a-4269-872f-d1377cb4ebea)

**Comment by eminence** [comment]
No comment on the style changes, but the change to open in a new tab LGTM

**Comment by sharkdp** [comment]
It's not a style change, this is just the dark mode of the website (if you set your browser to: "prefers theme: dark").

Or are you talking about the formatting changes in the HTML file? @hamirmahal can you please revert those?

**Comment by hamirmahal** [comment]
I put `style` because I consider preferring a link to open in a new tab, instead, a stylistic preference.

**Comment by hamirmahal** [comment]
@sharkdp I removed the formatting changes.

**Comment by eminence** [comment]
> It's not a style change

I meant whitespace style, sorry for the confusion

File: numbat-wasm/www/index.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
       <div id="terminal"></div>
       <p class="links"><a href="https://numbat.dev/doc/">Documentation</a> &bull; <a href="https://numbat.dev/doc/example-numbat_syntax.html">Syntax reference</a> &bull; <a href="https://numbat.dev/doc/web-usage.html">Commands and key bindings</a> &bull; <a href="https://github.com/sharkdp/numbat">Source code</a></p>
     </div>
-    <a href="https://github.com/sharkdp/numbat" class="github-corner" aria-label="View source on Github">
+    <a target="_blank" rel="noopener noreferrer" href="https://github.com/sharkdp/numbat" class="github-corner" aria-label="View source on Github">
```
Comment (sharkdp) [inline numbat-wasm/www/index.html:36]: `noopener` is standard for `target="_blank"` links. `noreferrer` is not needed.
```suggestion
    <a target="_blank" href="https://github.com/sharkdp/numbat" class="github-corner" aria-label="View source on Github">
```

**Review by sharkdp — APPROVED** [review]

**Pull request merged**